docs: reformat the documentation (#396)
ansys/ansys-sphinx-theme: doc/source/index.rst

Ansys Sphinx Theme documentation
################################

> Version: |version|

Reviewer: Why not have the version in the title like our other documentation? Having it here seems odd--and makes it stand out less.

> The Ansys Sphinx theme is a custom theme for Sphinx, a documentation generator
> used for creating project documentation from reStructuredText source files.
> 
> This theme is specifically tailored for documentation related to Ansys projects.
> It helps ensure consistency in the look and feel of documentation. Various
> useful extensions are included in the theme to make documentation more
> appealing and user-friendly.

Reviewer: ```suggestion
The Ansys Sphinx Theme is a custom Ansys-branded theme for use with Sphinx, a documentation generator
for creating project documentation from reStructuredText source files.

This theme is specifically tailored for documentation related to Ansys projects
helping to ensure consistency in its look and feel. Various useful extensions
are included in the theme to make documentation more appealing and user-friendly.
```

.. grid:: 2
    :gutter: 2 2 3 4

    .. grid-item-card:: Getting started :fa:`person-running`
        :link: getting-started
        :link-type: doc

>         Step-by-step guidelines on how to install the Ansys Sphinx theme.

Reviewer: ```suggestion
        Learn how to install the Ansys Sphinx Theme.
```

    .. grid-item-card:: User guide :fa:`book-open-reader`
        :link: user-guide
        :link-type: doc

>         Learn about the capabilitie, features, and key topics of this theme.

Reviewer: ```suggestion
        Learn how to use the capabilities and features of this theme.
```

    .. grid-item-card:: Examples :fa:`laptop-code`
        :link: examples
        :link-type: doc

>         Gallery of examples showing how third-party extensions integrate with this theme.

Reviewer: ```suggestion
        Explore examples that show how to integrate third-party extensions with this theme.
```
